### PR TITLE
Complete overhaul to the tracker

### PR DESCRIPTION
**Pull request by TheLX5**
This PR has the purpose to redesign and adapt the tracker for easier usage and include compatibility with the upcoming SMW v2 AP implementation.

Changes:
* Maps are now in a single image under a single tab
* There's a level image for every single level in the game
  * The levels have exact coordinates for every kind of check supported in them
* Added support for several SMW v2 features
  * Blocksanity
  * 3-Up Moons
  * Hidden 1-Ups
  * Bonus Blocks prizes
  * Special Zone Clear

### CODE REVIEW

File: super_mario_world_randomizer_porygone/manifest.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
     "name": "Super Mario World - Archipelago Tracker",
     "game_name": "Super Mario World",
     "platform": "Super Nintendo Entertainment System",
-    "package_version": "1.0.1",
+    "package_version": "2.0.0",
```
Comment (PoryGone) [inline super_mario_world_randomizer_porygone/manifest.json]:
```suggestion
    "package_version": "2.0.0b1",
```
I prefer to mark pre-release versions as incremental betas until the AP side gets merged and things stabilize.

**Comment by PoryGone** [comment]
Not a huge deal, but is there a reason that the subfolders of the `locations` folder were removed?

File: super_mario_world_randomizer_porygone/manifest.json (hunk 1 above)
Comment (TheLX5) [reply super_mario_world_randomizer_porygone/manifest.json]: Ah, alright.

**Comment by TheLX5** [comment]
> Not a huge deal, but is there a reason that the subfolders of the `locations` folder were removed?

My script just placed everything into the root folder, didn't bother with organization too much.

**Comment by TheLX5** [comment]
Added fixes for the things you reported to me via DMs. Also changed "Star World" instances to "Star Road" to match the apworld names.

I haven't gotten around updating the logic rules for the GSP wall in DP1, so I'm not gonna add those to the tracker yet.

**Pull request merged**